index.html

```diff
@@ -63,10 +63,34 @@
       gtag('js', new Date());
 
       gtag('config', 'G-YRG12HRZKB');
+
+      const signout = () => {
+        fetch('https://api.realdevsquad.com/auth/signout', {
+          method: 'GET',
+          credentials: 'include',
+        });
+      };
```

Comment:
can we put this in a separate js file and then add click event listener to our button and then call this function

Reply:
ok let me check that

Reply:
Won't it be ok if I do it in the same file but by using the add event listener?

Reply:
It would be much cleaner if we put it in js file

Reply:
ok. Let me try that

Reply:
As per the latest commit this issue is resolved